scripts/prepare

```diff
@@ -19,15 +19,16 @@ for f in packages/*; do
 			rm -rf ${f}/charts
 			mkdir -p ${f}/charts
 			if [[ $type == 'git' ]]; then
+				rm -rf /tmp/tmp-charts
 				mkdir -p /tmp/tmp-charts
 				git clone --depth=1 --no-tags $url /tmp/tmp-charts
 				pwd=$(pwd)
 				cd /tmp/tmp-charts
+				git fetch --tags
```

Comment:
This isn't necessary anymore right ? Since we are doing tgz and not git.

Reply:
good catch. removed!